tools/parallel-package-name.sh

```diff
@@ -7,13 +7,12 @@ NEW_ID=$1		# e.g. com.ichi2.anki.a
 NEW_NAME=$2		# e.g. AnkiDroid.A
 
 ROOT="AnkiDroid/src/main/"
-MANIFEST="AndroidManifest.xml"
 CONSTANTS="res/values/constants.xml"
 
 OLD_ID=com.ichi2.anki
 
 perl -pi -e "s/name=\"app_name\"(.*?)>.*?<\/string>/name=\"app_name\"\1>$NEW_NAME<\/string>/g" $ROOT$CONSTANTS
 perl -pi -e "s/applicationId \"$OLD_ID/applicationId \"$NEW_ID/g" AnkiDroid/build.gradle
-perl -pi -e "s/android:authorities=\"$OLD_ID/android:authorities=\"$NEW_ID/g" $ROOT$MANIFEST
-perl -pi -e "s/permission android:name=\"$OLD_ID.permission/permission android:name=\"$NEW_ID.permission/g" $ROOT$MANIFEST
-find $ROOT/res/xml -type f -exec perl -pi -e "s/android:targetPackage=\"$OLD_ID\"/android:targetPackage=\"$NEW_ID\"/g" {} \;
+# handle resValue
+# note: do not close the " after $OLD_ID so it handles "com...anki.debug" -> "com...anki.a.debug"
+perl -pi -e "s/\"applicationId\", \"$OLD_ID/\"applicationId\", \"$NEW_ID/g" AnkiDroid/build.gradle
```

Comment:
```suggestion
perl -pi -e "s/applicationId \".+?\"/applicationId \"$NEW_ID\"/g" AnkiDroid/build.gradle

```
if you are going to avoid the hardcoded strings, this change would be nice as well. 

This has the additional advantage of being able to re-run the script while having correct results. Before, if someone ran the script repeatedly, it would append the suffixes over the others (e.g. `com.ichi2.anki.a.b.a.a.b`)